fairseq/tasks/language_modeling.py

```diff
@@ -69,10 +66,6 @@ def add_args(parser):
                                  'If set to "eos", includes only one sentence per sample.')
         parser.add_argument('--tokens-per-sample', default=1024, type=int,
                             help='max number of tokens per sample for LM dataset')
-        parser.add_argument('--lazy-load', action='store_true',
-                            help='load the dataset lazily')
-        parser.add_argument('--raw-text', default=False, action='store_true',
-                            help='load raw text dataset')
```

Comment:
Let's provide backward compatibility for these. Add back these flags, change the description to indicate `[deprecated]` and add logic here: https://github.com/pytorch/fairseq/blob/master/fairseq/options.py#L111-L115.

Something like:
```
if getattr(args, 'raw_text', False):
    utils.deprecation_warning('--raw-text is deprecated, please use --dataset-impl=raw')
    args.dataset_impl = 'raw'
elif getattr(args, 'lazy_load', False):
    utils.deprecation_warning('--lazy-load is deprecated, please use --dataset-impl=lazy')
    args.dataset_impl = 'lazy'
```

Reply:
I still have to include these changes, I will work on this asap!